src/Classes/MyRadio/MyRadioFormField.php

```diff
@@ -784,7 +784,7 @@ private function convertTime($timeString)
          * YES, PHP IS RETARDED.
          */
         $timeString = str_replace('/', '-', $timeString);
-        $time = (int) strtotime($timeString) - date('Z');
+        $time = (int) strtotime($timeString);
```

Comment:
What this line does is confuse things. It removes an hour from the duration field on the allocate form, and combined with other things, makes the displayed requested time different from the actual requested time.